spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialConvolution.scala

```diff
@@ -70,7 +70,8 @@ class SpatialConvolution[T: ClassTag](
   val initGradBias: Tensor[T] = null,
   val withBias: Boolean = true,
   val format: DataFormat = DataFormat.NCHW
-)(implicit ev: TensorNumeric[T]) extends TensorModule[T] with Initializable {
+)(implicit ev: TensorNumeric[T]) extends TensorModule[T] with Initializable
+ {
```

Comment:
Need to change this file?
Are you moving compute output shape to new layers?

Reply:
The existing `computOutputShape` would not be touched.

Reply:
So for `SpatialConvolution.scala` and `SpatialDilatedConvolution.scala`, I don't think these files should be touched in this PR?